metron-analytics/metron-profiler-common/pom.xml

```diff
@@ -79,8 +79,26 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>${global_mockito_version}</version>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
Probably. I thought about doing it, and just didn't because I originally intended to go more module by module. Then I got really sick of find/replace module by module.

Particularly now that #1557 is in, it's probably safer to do this, but I'm a bit hesitant to do it here just because I'm slightly worried we'd see more of those issues and I mostly got lucky debugging that one.

It might be easier/cleaner to do this as a follow-on or some more general maven cleanup?